src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx

```diff
@@ -257,6 +263,8 @@ const ArtworksGrid: React.FC<ArtworksGridProps> = ({
     )
   }
 
+  const userDidScroll = artworks.length >= CREATE_ALERT_REMINDER_ARTWORK_THRESHOLD
```

Comment:
`userDidScroll` isn't very accurate. Maybe something like this is better:

```suggestion
  const displayAlertReminderMessage = artworks.length >= CREATE_ALERT_REMINDER_ARTWORK_THRESHOLD
```